src/i18n/en-events.ts

```diff
@@ -9,6 +9,7 @@ const eventsMessageDescriptors = {
       ${ColonyAndExtensionsEvents.TokensMinted} {{initiator} minted {amount} {tokenSymbol} to {recipient}}
       ${ColonyAndExtensionsEvents.DomainAdded} {{initiator} added Team: {fromDomain}}
       ${ColonyAndExtensionsEvents.ColonyUpgraded} {This colony has upgraded to {newVersion}}
+      ${ColonyAndExtensionsEvents.ColonyRoleSet} {{initiator} {role} permission in the {toDomain} team to {recipient}}
       other {{eventNameDecorated} emmited by {clientOrExtensionType}}
     }`,
   /*
```

Comment:
Please also add copy for the events:

![Screenshot from 2021-02-05 15-42-09](https://user-images.githubusercontent.com/1193222/107043279-74c4ab00-67cb-11eb-86a7-f0ce4e56fc85.png)

Let me know if you need help coming up with the correct copy